app/views/devise/registrations/edit.html.erb

```diff
@@ -1,54 +1,53 @@
-<div class="page page-dashboard">
-  <div class="wrapper wrapper--light">
-    <div class="container">
-      <div class="pagehead">
-        <div class="row">
-          <div class="col-sm-6">
-            <ol class="breadcrumb">
-              <li><%= link_to 'My Profile', user_path(current_user.id) %></li>
-              <li class="active">Edit</li>
-            </ol>
-          </div>
-          <div class="col-sm-6">
-            <div class="btn-group pull-right">
-              <%= link_to 'Back', :back, class: 'btn btn-default' %>
-            </div>
+<% provide(:page_class, 'page-dashboard') %>
```

Comment:
my read on all these massive changes are that you just added the `<% provide %>` and git is interpreting it as much bigger changes